roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: "StackStorm_{{ st2_pkg_repo }}"
+    baseurl: https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/el/{{ ansible_distribution_major_version }}/$basearch
+    repo_gpgcheck: yes
+    gpgkey: "https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/gpgkey"
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire: 300
+    gpgcheck: no
+    enabled: yes
+    sslverify: yes
+
+# Fixes "Failure talking to yum: Cannot retrieve repository metadata (repomd.xml) for repository: StackStorm_stable. Please verify its path and try again" when installing st2
+- name: Update ca-certificates package
+  become: yes
+  package:
+    name: ca-certificates
+    state: latest
```

Comment:
So the problem I had with this specific line (not other places where I made this linting error) was that unless I updated ca-certificates, I'd get this error when `st2mistral` runs:

`Failure talking to yum: Cannot retrieve repository metadata (repomd.xml) for repository: StackStorm_stable. Please verify its path and try again`

I just checked, and setting this to `present` instead of `latest` causes the error to come back up, as the package is already installed, so it doesn't do anything.